projects/samples/contests/robocup/controllers/referee/tests/launch_all_tests.sh

```diff
@@ -44,22 +44,22 @@ do
     folder=$(dirname ${test_file})
     test_log="${folder}/test.log"
     referee_log="${folder}/referee.log"
-    msg_prefix="[$NTH_TEST/${#TEST_FILES[@]}] $folder"
+    msg_prefix="[$NTH_TEST/${#TEST_FILES[@]}] ${folder:2}"
 
     if should_run_tests
     then
-        echo "$msg_prefix ..."
-        ./launch_test.sh ${folder} &> ${test_log}
+        printf '%-60.60srunning...' "$msg_prefix"
+        ./launch_test.sh ${folder} | tee ${test_log} > /dev/null
         cp ../log.txt ${referee_log}
     fi
     RESULT_LINE=$(awk '/TEST RESULTS/ { print $3 }' ${test_log})
     NB_SUCCESS=$(echo $RESULT_LINE | awk 'BEGIN { FS = "/" } ; { print $1 }')
     NB_TESTS=$(echo $RESULT_LINE | awk 'BEGIN { FS = "/" } ; { print $2 }')
     if [ $NB_SUCCESS -lt $NB_TESTS ]
     then
-        printf "$COLOR_RED$msg_prefix %s %2d/%2d$COLOR_RESET\n" FAIL $NB_SUCCESS $NB_TESTS
+        printf "\b\b\b\b\b\b\b\b\b\b$COLOR_RED%s %d/%d$COLOR_RESET  \n" FAIL $NB_SUCCESS $NB_TESTS
     else
-        printf "$COLOR_GREEN$msg_prefix %s %2d/%2d$COLOR_RESET\n" PASS $NB_SUCCESS $NB_TESTS
+        printf "\b\b\b\b\b\b\b\b\b\b$COLOR_GREEN%s %d/%d$COLOR_RESET  \n" PASS $NB_SUCCESS $NB_TESTS
```

Comment:
The `\b` approach has the problem that if you ctrl+c the script, it messes up the output. I personally wouldn't use the `\b` approach but I also think that this is not a big problem, just wanted to mention it. You can keep it as is, if you want.

Example:

```
[1/42] ball_holding/multi_robot_kid                         PASS 12/12
[2/42] ball_out/corner                                      PASS 4/4
[3/42] ball_out/goal_kick                                   running...^CWARNING: player: The process crashed some time after starting successfully.
WARNING: 'player' controller crashed.
WARNING: player: The process crashed some time after starting successfully.
WARNING: 'player' controller crashed.
FAIL 0/0
[4/42] ball_out/throw_in                                    running...^CWarning: QProcess: Destroyed while process ("python") is still running.
^C
```

Reply:
I see. In that case, I agree it is better to remove it. The `running...` message is not super useful, as the cursor is there, and so it could be removed, so that we don't need the  `\b`s anymore. I will do that on the PR.